concepts/Code_status/Readme.md

```diff
@@ -0,0 +1,9 @@
+The files here create a patient table that tracks the changes to the patient code status. The careplangeneral table was used instead of careplaneol because careplaneol had much less recorded patients, and the type of code status change was recorded in careplangeneral. There are two versions of the files: .Rmd and .sql. The Rmd version is the supported version, while the SQL version is not supported and was added just as a template.
```

Comment:
Instead of ignoring the information in `careplaneol`, could it be used to supplement the information in `careplangeneral`?

Reply:
It seems `careplaneol` is focused on discussion (e.g., family meeting), and contains little information about treatment limitations.